Emby.Server.Implementations/Library/UserManager.cs

```diff
@@ -366,6 +396,25 @@ private IAuthenticationProvider[] GetAuthenticationProviders(User user)
             return providers;
         }
 
+        private IPasswordResetProvider[] GetPasswordResetProviders(User user)
+        {
+            var passwordResetProviderId = user == null ? null : user.Policy.PasswordResetProviderId;
+
+            var providers = _passwordResetProviders.Where(i => i.IsEnabled).ToArray();
```

Comment:
I don't think this ToArray is strictly needed?

Reply:
Error	CS0266	Cannot implicitly convert type 'System.Collections.Generic.IEnumerable<MediaBrowser.Controller.Authentication.IPasswordResetProvider>' to 'MediaBrowser.Controller.Authentication.IPasswordResetProvider[]'. An explicit conversion exists (are you missing a cast?)

I get this if I remove it. I could try casting it, but an explicit convert seemed safer.